### ISSUE TITLE
Simple url logging only

### ISSUE THREAD

**Issue by Htcoulter** [mendhak/gpslogger#866]
I would like to know if it is possible to configure GPSLogger to only send current information to URL at the send frequently selected to either Log or Send? Not to send the log files. And if it is possible to have this send occur even if GPSLogger is not the top application, but is open?

**Comment by mendhak** [comment]
It's possible I'm misunderstanding the question.  The Custom URL function only sends current information to the target URL, it's sent at the log frequency (which is in seconds) not the send frequency (which is in minutes, meant for slow 'uploads').  There isn't a way to have the Custom URL work at the send frequency.  

For the second part - yes, GPSLogger should be able to work in the background.  Some users have reported that they need to go into the system's app permissions and set GPSLogger to always have access to location information.  As long as that's done it should run in the background.  You'll see a notification

**Comment by Htcoulter** [comment]
Perfect answer.. Works exactly now.  great software

**Comment by Htcoulter** [comment]
2nd phone.. Install OK, Logging on start OK. Settings exactly same as first phone, but logger does not run location fix and log on time interval selected (or at all).. All permissions same on both phones. Where should I look for the reason?

**Comment by mendhak** [comment]
In the app settings, have you allowed the app to get location all the time (not just while using the app)?  It's a setting you have to go into the app settings for.  

![image](https://user-images.githubusercontent.com/746276/116614570-00188d00-a932-11eb-922d-b1f6e04ffb9e.png)

**Comment by Htcoulter** [comment]
Yes.. this is enabled. The problem can clearly be seen when viewing the
logging tab. One phone polls the gps and send to url every 30 seconds as I
set to, the other phone sends when the app is opened once.. then not again.
I have side by side checked all settings are the same in both phones

**Comment by mendhak** [comment]
Hm that's the only reason I can think of, that it's not running in the background due to a missing permission.  But if you're saying that permission is set as well... 

Is the 2nd phone's location setting set to use network or satellite?  

Any accuracy filters, any distance filters?  

What models are the two phones?

I don't think it's related but also have a look at how aggressive the battery management is:  https://dontkillmyapp.com/

**Comment by Htcoulter** [comment]
I'll look at the battery management. Also, I set up 14 more phones and all
installed and worked fine, except two.. one could not get to the gps (did
not have time to dig in deeper) and the other was the same as phone we are
discussing.. so to try and be clearer..

Open app
Select start logging
Switch view to logging view
First gps point is located
This point is sent to url correctly
Then all activity stops, no more points logged..

On other devices, after 30 seconds (setting) next point is located and
transmitted to url.

**Comment by mendhak** [comment]
Thanks for clarifying, now I see the problem isn't what I thought.  You're saying that the logging stops even if the app is in the _foreground_.  

Can you try to create a debug log as shown here: https://gpslogger.app/#troubleshooting

And email that to me, `gpslogger@mendhak.com`, or if you want you can just post it here.  I'm hoping there'll be some kind of error or problem highlighted that will give a clue.

**Comment by edukmattos** [comment]
Hi @mendhak 
you said: "It's possible I'm misunderstanding the question. The Custom URL function only sends current information to the target URL, it's sent at the log frequency (which is in seconds) not the send frequency (which is in minutes, meant for slow 'uploads'). There isn't a way to have the Custom URL work at the send frequency."

Im having the problem I set the interval to 30s and the app send every second.

**Comment by mendhak** [comment]
> There isn't a way to have the Custom URL work at the send frequency.

In v119 the ability to 'send' Custom URL at an interval has been added, so now it is possible.  It works by enabling CSV logging and every 60 minutes (default) it reads each line and makes the HTTP request.  

This should help with some of the issues around Custom URL reliability
